docs/narr/sessions.rst

```diff
@@ -85,32 +85,49 @@ This is a stricter contract than the previous requirement that all objects be pi
 This is a backward-incompatible change.
 Previously, if a client-side session implementation was compromised, it left the application vulnerable to remote code execution attacks using specially-crafted sessions that execute code when deserialized.
 
+Please reference the following tickets if detailed information on these changes is needed:
```

Comment:
Is this something we've done in the past, having our narrative documentation link to issue tickets?

Reply:
I don't think I've seen that in this project per-se, but a handful of other projects do this when there is some sort of extended discussion detailing a security related issue, but it's just not worth repeating all of that stuff within the documentation itself.

Reply:
Bert's correct. I just scanned narrative docs, and there are no direct links to a specific issue or PR. As I consider it, it makes sense to not add a maintenance burden, especially if things change again in the future.

Let's just fall back to the entries in `CHANGES.rst` and call it soup.